### ISSUE TITLE
Feature request: include hover links for all titles and subtitles

### ISSUE THREAD

**Issue by grokky1** [aspnet/Docs#4162]
I hope this is the correct place to discuss.

Many pages have a floating TOC on the right hand side, which is great. But often it doesn't contain all the document's sections and titles and subtitles.

It would be great if every section/header/title/subtitle/etc had a hover link. Exactly like the wiki pages here on GitHub (or on readthedocs.io) - there's a small link icon to the left or right of each title (h1/h2/h3/etc), which shows when you hover over that title.

I've checked the source on many pages, and titles/subtitles *already* define an `id` attribute. So it's really just a matter of exposing it via a hover icon.

**Comment by scottaddie** [comment]
@EisenbergEffect Thoughts?

**Comment by EisenbergEffect** [comment]
Hey @grokky1 Thanks for sharing this idea! This is something that we have tracked in our docs experience feature backlog. We've actually even got some early design comps for what we want the experience to be. I'm not sure if/when we'll be able to ship this feature, as there are many things we're working on, many of which take a higher priority at the moment. However, this is something we're considering and I do hope to make this improvement in the future.

I'll close this issue here since it's not an ASP.NET issue, but an issue for docs.micrsoft.com. We've also got this fully tracked in our own backlog.

Thanks again for the thoughtful request @grokky1 !
